proposals/lip-0063.md

```diff
@@ -365,19 +375,20 @@ def addTokenModuleEntry():
     tokenObj.supplySubstore = createSupplySubstoreArray()
     tokenObj.escrowSubstore = []
     tokenObj.availableLocalIDSubstore = {}
+    tokenObj.supportedTokensSubstore = [{"chainID": MAINCHAIN_ID, "supportedTokenIDs": [TOKEN_ID_LSK]}]
```

Comment:
Checking the function `isTokenSupported` in the Token module and rationale section, I think this is not needed (although it is also not incorrect).  Not sure about this one.

Reply:
Agree, removed. I think, we should not put LSK into supported tokens substore.